.github/workflows/packages.yaml

```diff
@@ -46,8 +53,10 @@ jobs:
         docker run --rm \
           -v "$GITHUB_WORKSPACE:/src" \
           -e "ARCH=$ARCH" \
+          -e "OS=$OS" \
           -e "PACKAGE_VERSION=$PACKAGE_VERSION" \
           -e "PACKAGE_RELEASE=$PACKAGE_RELEASE" \
+          --privileged \
```

Comment:
Why does it need `--privileged` ?

Reply:
Its required for the mariner toolkit to build here. have tried it without privileged an cannot get that to work in this environment.

Reply:
Do you know which command requires privilege and why? It's not a blocker. I'm just curious.

Reply:
I believe it is when we make the RPM file so it would be the "make build-packages" command in packages.sh. it has been a couple months since testing that specifically so that might not be right.

Reply:
I see a bunch of chrooting going on, so it's probably that.